tests/src/test/scala/whisk/core/containerpool/test/ContainerPoolTests.scala

```diff
@@ -78,8 +81,26 @@ class ContainerPoolTests extends TestKit(ActorSystem("ContainerPool"))
         ActivationId(),
         invocationNamespace.toPath,
         None)
+    val messageb = ActivationMessage(
```

Comment:
Can we give those more meaningful names? Like `differentNamespaceMessage` and `differentMessage`?